### PR TITLE
Create comments

### PR DESCRIPTION
**Pull request by kvh8899**
- a user can now delete there own comments
- a user can now create comments
- a user can no longer delete comments they do not own
- after deleting/creating comment, page is reloaded with sidebar visible
- comments are now rendered from newest to oldest
- fixed a bug in the mixing that was not reading posts correctly
- following feed on the home page now renders followed posts
- fixed a display issue on the home page for a button that was not rendering correctly when user has 0 users following
- deleted unused comments in css files

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -39,7 +40,8 @@ store.sync();
 
 app.use('/', indexRouter);
 app.use('/stories', storiesRouter);
-
+app.use('/comments',commRouter);
+app.use
```
Comment (justinrusso) [inline app.js]: This can be removed

File: routes/stories.js [hunk 2]
```diff
@@ -117,7 +117,10 @@ router.get(
         },
       });
     }
-
+    //sort comments by newest
+    story.Comments.sort((b,a) => {
+     return a.createdAt.getTime() - b.createdAt.getTime();
+    });
```
Comment (justinrusso) [inline routes/stories.js]: This should be do-able in the query. Something like:

```js
const story = await Post.findByPk(storyId, {
  include: [
    { model: Comment, include: [CommentLike, User] },
    PostLike,
    { model: User, include: [{ model: User, as: "Followers" }] },
  ],
  order: [
    [Post, Comment, 'createdAt', 'DESC'],
  ],
});
```

File: views/fixed-footer.pug [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        li
+            a(href="") Blog 
+        li
+            a(href="") Careers
+        li
+            a(href="") Privacy
+        li
+            a(href="") Terms 
+        li
+            a(href="") About
```
Comment (justinrusso) [inline views/fixed-footer.pug]: This file shouldn't be here - I assume this might be from a bad merge?

**Review by justinrusso — CHANGES_REQUESTED** [review]

File: routes/stories.js (hunk 2 above)
Comment (kvh8899) [reply routes/stories.js:123]: this query breaks the rest of the code

**Review by justinrusso — APPROVED** [review]
Looks good!

**Pull request merged**